### DISCUSSION TITLE
Missing actions

### DISCUSSION THREAD

**Discussion by bsubercaseaux** [alshedivat/al-folio#419] [Q&A]
I just followed all instructions (hope I didn't miss anything), and my website is not updating when I push to master. I now realized that on my fork (https://github.com/bsubercaseaux/bsubercaseaux.github.io) there are no actions, so of course the `gh-pages` branch is not updating.

How do I add the required actions?

**Comment by rohandebsarkar** [comment]
Please go to the repository settings and under `Actions` make sure `Allow all actions` are enabled.
And also make sure under `Pages` the source for GitHub Pages is set to `gh-pages`.

**Reply by bsubercaseaux** [reply]
I had both options set that way, but I still don't see anything.

![image](https://user-images.githubusercontent.com/16748865/135906183-5a79ceab-1185-4577-8c88-bb1bf5dc6d86.png)

**Reply by rohandebsarkar** [reply]
Did you push the changes to `master` after enabling GitHub Actions?
